extensions/amp-user-notification/0.1/amp-user-notification.css

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-user-notification {
+  visibility: hidden;
+}
+
+.amp-user-notification {
+  position: fixed;
```

Comment:
Alternatively we need to put the dialogs into a single position:fixed
container.

On Mon, Dec 21, 2015 at 9:16 AM erwin mombay <notifications@github.com>
wrote:

> In extensions/amp-user-notification/0.1/amp-user-notification.css
> <https://github.com/ampproject/amphtml/pull/1178#discussion_r48167623>:
>
> > + *
> > + *      http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS-IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +amp-user-notification {
> > +  visibility: hidden;
> > +}
> > +
> > +.amp-user-notification {
> > +  position: fixed;
>
> if we do position: fixed !important and an implementor of a page uses 2
> amp-user-notification elements, they are going to stack on each other,
> are we ok with that? (do we queue up the notifications?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amphtml/pull/1178/files#r48167623>.
>